source/incident-collaboration/running-playbooks.rst

```diff
@@ -87,7 +87,7 @@ Playbook run members can end an incident using the ``/playbook end`` slash comma
 5. Enter a message with additional details.
 6. Select **Update Status**.
 
-Ending a playbook run signals to all members of the channel that the playbook run is complete. Members of the team can continue to post in the channel, mark tasks as complete, and change the owner if needed.
+Ending a playbook runs signals to all members of the channel that the playbook run is complete. Members of the team can continue to post in the channel, mark tasks as complete, and change the owner if needed.
```

Comment:
In this context, it's the run (of type playbook) itself that signals something to all members of the channel; therefore, the term ``playbook run`` is correct.